packages/components/src/dropdown-menu/style.scss

```diff
@@ -82,6 +82,10 @@
 		margin-top: 0;
 		border-top: $border-width solid $light-gray-secondary;
 		padding: $grid-unit-15;
+
+		.is-alternate & {
```

Comment:
Is this class still in use?

Reply:
Hey @gziolo . Still in `WIP` status. I will make more changes tomorrow :)

Reply:
Yes this class is still in use. `is-alternate` is a prop that should be applied to all popovers inside the editing canvas, so as to provide the same dark borders as the block UI itself.